### PR TITLE
Improve script for local CCD Web boot

### PR DESCRIPTION
**Pull request by doncem**
### JIRA link (if applicable) ###

[Fix ccd docker for local testing](https://tools.hmcts.net/jira/browse/BPS-620)

### Change description ###

- More README:
  - mention caseworkers
  - troubleshooting section
- DEV account selection in case your user is enlisted in such subscription
- checking whether you are logged in in `AZ ACR`
  - less info logs when you run scripts and you are already logged in
  - longer run (few seconds) if you are not logged in

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +95,27 @@ $ ./bin/upload-ccd-spreadsheet.sh -v
...
+##### CCD Web is up and running, but cannot log in
+
+There can be multiple reasons including core breaking changes introduced by services enlisted in `docker-compose.yml` file.
+
+First course of actions is to check whether CCD definition got imported successfully as it is creating user profiles which are mandatory for login
+
+The following are suggestions of possible culprits:
+
+- role mismatch between idam and ccd. Check both `ccd-importer` and `idam-importer`
+- debug down the line from `ccd-case-management-web`
```
Comment (nitinprabhu) [inline README.md]: What do you mean by this? do you mean check docker logs for ccd-case-management-web ?

File: README.md [hunk 2]
```diff
@@ -92,6 +95,27 @@ $ ./bin/upload-ccd-spreadsheet.sh -v
...
+IdAM API takes long time to boot up.
+This causes incomplete docker setup.
+Inspect `docker ps -a` idam container, wait for completion, then run `./bin/start-ccd-web.sh` again.
+In all occasions never experienced a failure afterwards
+
+##### CCD Web is up and running, but cannot log in
+
+There can be multiple reasons including core breaking changes introduced by services enlisted in `docker-compose.yml` file.
+
+First course of actions is to check whether CCD definition got imported successfully as it is creating user profiles which are mandatory for login
```
Comment (nitinprabhu) [inline README.md]: actions ->action ?

File: README.md [hunk 3]
```diff
@@ -92,6 +95,27 @@ $ ./bin/upload-ccd-spreadsheet.sh -v
...
+
+The following are suggestions of possible culprits:
+
+- role mismatch between idam and ccd. Check both `ccd-importer` and `idam-importer`
```
Comment (nitinprabhu) [inline README.md]: you mean container logs here I assume ?

Comment (doncem) [reply README.md]: This whole section is about local ccd web setup in docker. Of course it'll be about containers/images

File: README.md (hunk 1 above)
Comment (doncem) [reply README.md]: same here. section is about docker setup

**Pull request merged**